samples/99_zil_interpreter/tests/parser_tests_basic.rb

```diff
@@ -0,0 +1,464 @@
+# test basic SETG command with STRING and FIX
+# -----------------------------------------
+def test_parser_01(args, assert)
+  source = <<-ZIL
+<SETG ZORK-NUMBER 1>
+  ZIL
+
+  parser = Parser.new
+  parser.parse_string(args, source)
+  parsed = parser.expressions[0]
+
+  expected = Syntax::Form.new(
+    :SETG,
+    :"ZORK-NUMBER",
+    1)
+
+  assert.equal! parsed, expected
+end
+
+# test character
+# -----------------------------------------
+def test_parser_02(args, assert)
+  # need to escape backslash in HEREDOC to get correct result
+  source = <<-ZIL
+<STRING !\\" !,WBREAKS>
+  ZIL
+
+  parser = Parser.new
+  parser.parse_string(args, source)
+  parsed = parser.expressions[0]
+
+  expected = Syntax::Form.new(
+    :STRING,
+    '"',
+    :"!,WBREAKS"
```

Comment:
This should probably be

```suggestion
    Syntax::Segment.new(Syntax::Form.new(:GVAL, :WBREAKS))
```

Same in the next test....

Reply:
Fixed.